src/main/java/uk/gov/justice/digital/service/HiveTableService.java

```diff
@@ -63,18 +64,19 @@ public Set<ImmutablePair<String, String>> replaceTables(ImmutableSet<ImmutablePa
             try {
                 validateTableName(hiveTableName);
                 StructType schema = sourceReference.getSchema();
+                SourceReference.PrimaryKey primaryKey = sourceReference.getPrimaryKey();
 
                 String rawArchivePath = createValidatedPath(jobArguments.getRawArchiveS3Path(), sourceName, tableName);
-                StructType rawSchema = withMetadataFields(schema);
-                replaceParquetInputTables(jobArguments.getRawArchiveDatabase(), hiveTableName, rawArchivePath, rawSchema);
+                StructType rawSchema = withScdFields(withMetadataFields(schema));
```

Comment:
Adds the columns `checkpoint_col` and `update_type` to the schema contract when creating the Hive tables for the raw-archive.